packages/components/src/mobile/bottom-sheet/range-text-input.native.js

```diff
@@ -218,7 +219,7 @@ class RangeTextInput extends Component {
 							returnKeyType="done"
 							numberOfLines={ 1 }
 							defaultValue={ `${ inputValue }` }
-							value={ inputValue }
+							value={ inputValue.toString() }
```

Comment:
This change concerns me a bit. I'd like to do some explicit testing of this component to verify no regressions are introduced.

Reply:
Tested `RangeTextInput` for Column block on iOS and Android. Appeared to function as expected. 👍🏻